extensions/amp-image-lightbox/0.1/amp-image-lightbox.js

```diff
@@ -26,6 +26,7 @@ import {
   TapzoomRecognizer,
 } from '../../../src/gesture-recognizers';
 import {Gestures} from '../../../src/gesture';
+//import {SwipeYRecognizer} from '../../../src/gesture-recognizers';
```

Comment:
@HeidiW Although this will enable browser's default pinch-zoom, it would zoom the whole page instead of just the image.

We have an internal component called `amp-image-viewer` which has pinch and double-tap functionality already built in. Idea is to integrate that component with `amp-image-lightbox` similar to how `amp-gallery-lighbox` integrated it in https://github.com/ampproject/amphtml/pull/12219/files. This PR would be a great reference for the works that needs to be done here.

This is one of our more involved good first issues, me and @cathyxz would be happy to answer questions on Slack (@cathyxz is traveling over the next two weeks and might be slow to respond)

Reply:
Thank you so much for not letting this slip @aghassemi! 

Yes, what @aghassemi described above would be the ideal implementation. The simpler implementation that I originally thought of here as a GFI, was to basically take what we had in `<amp-image-viewer>` regarding pinch zoom and implement that here in `<amp-image-lightbox>`. 

Basically: 

https://github.com/ampproject/amphtml/blob/e19b9b9519a908ec956a6f16876ebd63c0697674/extensions/amp-image-viewer/0.1/amp-image-viewer.js#L638-L660

and 
https://github.com/ampproject/amphtml/blob/e19b9b9519a908ec956a6f16876ebd63c0697674/extensions/amp-image-viewer/0.1/amp-image-viewer.js#L436-L444

A bit of refactoring is definitely overdue on this component, but would definitely be a bit of a challenge for a first time PR. If you're interested, we could start with the simpler approach and work up to integrating `<amp-image-viewer>`. @HeidiW what do you think?

Reply:
So the original implementation that I had in mind when filing this issue was the simpler approach of taking what we had already implemented in `<amp-image-viewer>` and backfilling it here. Specifically: 
https://github.com/ampproject/amphtml/blob/e19b9b9519a908ec956a6f16876ebd63c0697674/extensions/amp-image-viewer/0.1/amp-image-viewer.js#L436-L444
and
https://github.com/ampproject/amphtml/blob/e19b9b9519a908ec956a6f16876ebd63c0697674/extensions/amp-image-viewer/0.1/amp-image-viewer.js#L611-L623
and 
https://github.com/ampproject/amphtml/blob/e19b9b9519a908ec956a6f16876ebd63c0697674/extensions/amp-image-viewer/0.1/amp-image-viewer.js#L638-L660
What @aghassemi outlined here is definitely the better way to go, but is probably a bit challenging for a first ever PR.

Reply:
So the original implementation that I had in mind when filing this issue was the simpler approach of taking what we had already implemented in `<amp-image-viewer>` and backfilling it here. Specifically: 
https://github.com/ampproject/amphtml/blob/e19b9b9519a908ec956a6f16876ebd63c0697674/extensions/amp-image-viewer/0.1/amp-image-viewer.js#L436-L444
and
https://github.com/ampproject/amphtml/blob/e19b9b9519a908ec956a6f16876ebd63c0697674/extensions/amp-image-viewer/0.1/amp-image-viewer.js#L611-L623
and 
https://github.com/ampproject/amphtml/blob/e19b9b9519a908ec956a6f16876ebd63c0697674/extensions/amp-image-viewer/0.1/amp-image-viewer.js#L638-L660
What @aghassemi outlined here is definitely the better way to go, but is probably a bit challenging for a first ever PR. 

Thank you so much @aghassemi for not letting this slip through the cracks.